static/src/javascripts/projects/common/modules/article/chapters.js

```diff
@@ -0,0 +1,24 @@
+define([
+    'bean',
+    'common/utils/$'
+], function (
+    bean,
+    $
+) {
+    return {
+        init: function () {
+            this.bindEvents();
+        },
+
+        bindEvents: function () {
+            bean.on(document.body, 'click', '.js-toggle-chapters', function (e) {
+                e.preventDefault();
```

Comment:
Because I did it in the past it doesn't mean its good solution :-) It depends on the context of the link. Do we need it to behave like a link without JS (should it take user somewhere without JS?) or should it behave like a button. Maybe the use of button element will be more semantic if it should behave as button.